### PR TITLE
INS0992 0992

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Two new records, one for the Übersee Museum on Fehmarn, another for the Serbian National Library, where up to 4 or 5 Ethiopic mss have turned up.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/INS0991Serbian.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="DN" when="2022-12-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place sameAs="wd:Q111718320" type="academicInstitution library" subtype="institution">
+                    <placeName xml:id="n1" xml:lang="en">National Library of Serbia</placeName>
+                    <placeName xml:id="n2" xml:lang="sr">Народна библиотека Србије</placeName>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/INS0991Serbian.xml:49]: I thought the same, but when I include  `<language ident="sr">Gǝʿǝz</language> ` the system reports a mistake.

File: new/INS0992UeberseeFehmarn.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <place type="museum" subtype="institution">
+                    <placeName xml:lang="ger">Museum Übersee</placeName>
+                    <country ref="wd:Q183"/>               
+                    <settlement type="city" ref="wd:Q3172"/>
+                    <location>
+                        <geo>54.422333 11.188998</geo>
+                    </location>
+                </place>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="INS0992UeberseeFehmarn" passive="MUFet01"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/INS0992UeberseeFehmarn.xml:57]: ok

File: new/INS0991Serbian.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/INS0991Serbian.xml:49]: Of course if sr is not in the taxonomy.

[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/INS0991Serbian.xml:49]: Does it mean, one of the files in "Schema" folder should be changed? I see there is a list of languages in tei-betamasaheft.xml file. Should "sr" be included there?

[the remark this answers is not in the document]
Comment (thea-m) [reply new/INS0991Serbian.xml:49]: yes, here is the guide: https://betamasaheft.eu/Guidelines/?id=editingSchema
(contrary to what I said first, the taxonomy does not need to be updated)

**Pull request merged**